src/query/query_hash_scan.c

```diff
@@ -1387,16 +1388,12 @@ fhs_initialize_bucket_new_page (THREAD_ENTRY * thread_p, PAGE_PTR page_p, void *
 static int
 fhs_initialize_dk_bucket_new_page (THREAD_ENTRY * thread_p, PAGE_PTR page_p, void *args)
 {
```

Comment:
the argument "args", it looks not necessary.

Reply:
I will keep it for future use.